src/crimson/os/seastore/cached_extent.h

```diff
@@ -550,7 +567,8 @@ class ExtentIndex {
 
   void erase(CachedExtent &extent) {
     assert(extent.parent_index);
-    auto erased = extent_index.erase(extent);
+    auto erased = extent_index.erase(
+      extent_index.s_iterator_to(extent));
```

Comment:
Just out of curiosity: why need to change to `extent_index.s_iterator_to(extent)`?